docs/en/siem/tune-anomaly-results.asciidoc

```diff
@@ -99,3 +102,40 @@ image::start-job-window.png[]
 . Click *Start*.
 +
 After a while, results will start to appear on the *Anomaly Explorer* page.
+
+[[define-rule-threshold]]
+==== Define an anomaly threshold for a job
+
+Certain jobs use a high count function to look for unusual spikes in 
+process events. Some processes cause a burst of activity as a normal part of 
+their function, such as automation and housekeeping jobs running on server 
+fleets. However, sometimes a high-delta event count is unlikely to be the 
+result of routine behavior. In these cases, defining a minimum threshold for 
+when a high event count is considered an anomaly can be applied to the {ml} 
+jobs.
+
+Depending on your anomaly detection results, it may be useful to define a 
+minimum event count threshold for the `packetbeat_dns_tunneling` job:
+
+
+. Go to *Machine Learning* -> *Anomaly Detection* -> *Anomaly Explorer*.
+. Navigate to the job results for the `packetbeat_dns_tunneling` job. If the 
+job results are not listed, click *Edit job selection* and select 
+`packetbeat_dns_tunneling`.
+. In the *actions* column, click the gear icon and then select
+_Configure rules_.
++
+The *Create Rules* windows is displayed.
```

Comment:
"windows" should be "window" 

General comment: I usually use "appears" instead of "is displayed" to keep active voice, but that's up to you.